modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheUtils.java

```diff
@@ -139,6 +139,9 @@
     /** */
     public static final int UNDEFINED_CACHE_ID = 0;
 
+    /** */
+    public static final int MAX_FILE_NAME_LENGTH = 235;
```

Comment:
I guess should be renamed to `CACHE_NAME_MAX_LENGTH`

Reply:
done